File: Rest-Api-V3/market-data-public/assets/assets.py
import requests

def fetch_assets_data():
    asset = 'BTC'
    url = f'https://api.ox.fun/v3/assets?asset={asset}'

    try:
        response = requests.get(url)
        response.raise_for_status()  # Raise an error for bad responses
        print('Assets Data:', response.json())
    except requests.exceptions.RequestException as error:
        print('Error fetching assets data:', error)

fetch_assets_data()

File: Rest-Api-V3/market-data-public/depth/assets/assets.js
const axios = require('axios');

async function fetchMarketData() {
  const asset = 'BTC';
  const url = `https://api.ox.fun/v3/assets?asset=${asset}`;

  try {
    const response = await axios.get(url);
    console.log('Market Data:', response.data);
  } catch (error) {
    console.error('Error fetching market data:', error.response ? error.response.data : error.message);
  }
}

fetchMarketData();